js/custom-post-parents-admin.js

```diff
@@ -1,61 +1,45 @@
+
 ;(function ($) {
     $(function () {
         var $ul = $('.custom-post-parents-selectedlist');
-        var $adders = $('.custom-post-parents-selectlist-add');
-
-        $ul.on('click', '.ntdelbutton', function () {
+        $ul.on('click','.ntdelbutton',function(){
+            var id_option=$(this).attr("name");
+            var section=$(this).attr("data-section");
+            $("#cpt-f-"+section+" >option[id='"+id_option+"']").show();
             $(this).parents('span').remove();
         });
+        var it=$("<option>").attr("name","custom-post-parents-post_types").attr('id','defecto');
+        it.text(data.first_option_select);
+        $('.custom-post-parents-selectlist-add').append(it)
+        $.each(data.lookup,function(i,v){
+            var it=$("<option>").attr("name","custom-post-parents-post_types").attr("id",v);
+            it.text(v);
+            $('.custom-post-parents-selectlist-add').append(it);
+        })
 
-        var cache = {};
-        $adders.autocomplete({
-            minLength: 0,
-            source: function( request, response ) {
-                var term = request.term;
-                if ( term in cache ) {
-                    response( cache[ term ] );
-                    return;
-                }
-                $.post(ajaxurl, {
-                        'action': 'custom-post-parents_search_post_types'
-                    },
-                    function (result) {
-                        $.each(result.data.lookup, function (index, item) {
-                            cache[index] = {value: index, label: item};
-                        });
-                        response( cache );
-                    }
-                );
-            },
-            select: function( event, ui ) {
-                var label = ui.item.label;
-                var type = ui.item.value;
-                var $input = $(event.target);
-                var section = $input.data('section');
-                var selectedElements = $('input[name="custom-post-parents-post_types['+section+'][]"]');
-                var currentVals = [];
-                $.each(selectedElements, function () {
+        $('.custom-post-parents-selectlist-add').change(function(event){
+            var target=event.target;
+            var section=target.getAttribute("data-section")
```

Comment:
He modificado lo que me has comentado y he eleminado la variable target, ya que con la modificación no se utiliza. Queda de la siguiente manera:

        $('.custom-post-parents-selectlist-add').change(function(event){
            var section=$(this).data("section")